deploy-odh-quicklab.sh

```diff
@@ -0,0 +1,24 @@
+API=upi-0.mptest.lab.upshift.rdu2.redhat.com:6443
+
+EMP="\e[1;4m"
+NORMAL="\e[0m"
+
+echo -e "${EMP}Checking that the ODH operator is present in the Marketplace${NORMAL}"
+if ! oc get packagemanifests/opendatahub-operator -n openshift-marketplace >/dev/null; then
+  echo -e "${EMP}ODH is missing in the operator manifests --> aborting${NORMAL}"
+  exit 1
+fi
+
+oc new-project odh-operator
+oc new-project odh-deployment
```

Comment:
Can we make this future proof? Maybe grep the namespaces from the `example/odh-*` resources?

Reply:
added